packages/components/psammead-navigation/src/Dropdown/index.jsx

```diff
@@ -0,0 +1,217 @@
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+import { getPica } from '@bbc/gel-foundations/typography';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+
+const getStyles = dir => {
+  const direction = dir === 'ltr' ? 'left' : 'right';
+  return `border-${direction}: ${GEL_SPACING_HLF} solid ${C_WHITE};
+          padding-${direction}: ${GEL_SPACING}`;
+};
+
+const DropdownWrapper = styled.div`
+  background-color: ${C_EBON};
+
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    display: none;
+  }
+`;
+
+const CrossButton = styled.button`
+  background-color: ${C_EBON};
+  width: 2.75rem;
+  height: 2.75rem;
+  padding: 0;
+  margin: 0;
+  border: 0;
+
+  &:focus {
+    box-shadow: inset 0 0 0 0.25rem ${C_WHITE};
+  }
+`;
+
+const DropdownUl = styled.ul`
+  list-style-type: none;
+  margin: 0;
+  padding-right: ${GEL_SPACING};
+  padding-left: ${GEL_SPACING};
+  border-bottom: 0.125rem solid ${C_SHADOW};
+`;
+
+const DropdownLi = styled.li`
+  padding-top: 0.75rem;
+  padding-bottom: 0.75rem;
+  border-bottom: 0.0625rem solid ${C_SHADOW};
+
+  &:last-child {
+    padding-bottom: ${GEL_SPACING_DBL};
+    border: 0;
+  }
+`;
+
+const StyledDropdownLink = styled.a`
+  ${({ script }) => script && getPica(script)};
+  ${({ service }) => service && getSansRegular(service)}
+  color: ${C_WHITE};
+  text-decoration: none;
+
+  &:focus {
+    text-decoration: underline;
+  }
+`;
+
+const StyledCurrentLink = styled.span`
+  ${({ dir }) => getStyles(dir)}
+`;
+
+export const DropdownNavigationLi = ({
+  children,
+  script,
+  currentPageText,
+  active,
+  service,
+  url,
+  dir,
+}) => (
+  <DropdownLi role="listitem">
+    {active && currentPageText ? (
+      <StyledDropdownLink script={script} service={service} href={url}>
+        <StyledCurrentLink
+          dir={dir}
+          // eslint-disable-next-line jsx-a11y/aria-role
+          role="text"
+        >
+          <VisuallyHiddenText>{currentPageText}</VisuallyHiddenText>
+          {children}
+        </StyledCurrentLink>
+      </StyledDropdownLink>
+    ) : (
+      <StyledDropdownLink script={script} service={service} href={url}>
+        {children}
+      </StyledDropdownLink>
+    )}
+  </DropdownLi>
+);
+
+export const AmpDropdown = ({ announcedText, onClose, children }) => (
+  <DropdownWrapper>
+    <CrossButton
+      aria-label={announcedText}
+      on={onClose}
+      // eslint-disable-next-line react/jsx-boolean-value
+      aria-expanded={true}
+    >
+      {navigationIcons.cross}
+    </CrossButton>
+    <DropdownUl role="list">{children}</DropdownUl>
+  </DropdownWrapper>
+);
+
+export const CanonicalDropdown = ({ announcedText, onClose, children }) => (
+  <DropdownWrapper>
+    <CrossButton
+      aria-label={announcedText}
+      onClick={onClose}
+      // eslint-disable-next-line react/jsx-boolean-value
+      aria-expanded={true}
```

Comment:
Quick question about using 'true' and 'false' with aria tags - should it be `aria-expanded` (a boolean attribute as it is here) or `aria-expanded="true"` (an attribute with 'true' as a string)?

Reply:
I'm just wondering as some properties appear to be able to be true/false/undefined

Reply:
Good point. On [React Accessibility](https://reactjs.org/docs/accessibility.html) page they use strings with any `aria` attributes, so I guess that is the correct option. Will change it now